widgets/contact/styles/default.less

```diff
@@ -56,12 +56,42 @@
 		}
 	}
 
+	@field_font_family: default;
+	@field_font_weight: default;
+	@field_font_size: default;
+	@field_font_color: default;
+
+	@field_background: default;
+
+	@field_margin: 0;
+	@field_padding: 0;
+	@field_border_radius: 0;
+	@field_border_color: default;
+	@field_border_width: default;
+	@field_border_style: default;
+	@field_height: 0;
```

Comment:
These should have `default` as the default value. Using `0` could unintentionally overwrite existing users' styles.